packages/babel-register/test/index.js

```diff
@@ -69,4 +75,32 @@ describe("@babel/register", function() {
       })
       .to.throw(SyntaxError);
   });
+
+  it("Does not install source map support if asked not to", () => {
```

Comment:
capitalizing the first letter of all these `it` blocks is inconsistent with the rest of the tests in this file.